content/millennium/r4/scheduling/appointment.md

```diff
@@ -66,23 +66,25 @@ _Implementation Notes_
 
 ### Parameters
 
- Name           | Required?                                                | Type          | Description
-----------------|----------------------------------------------------------|-------------------------------------------------------------------------------------------------
- `_id`          | Yes, or one of `patient`, `practitioner`, or `location`. | [`token`]     | The logical resource id associated with the Appointment. Example: `7890`
- `date`         | Yes when using `patient`, `practitioner`, or `location`. | [`date`]      | The Appointment date time with offset. Example: `2019-06-07T22:22:16.270Z`
- `patient`      | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Patient references. Example: `1234`
- `practitioner` | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Practitioner references. Example: `4567`
- `location`     | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Location references. Example: `6789`
- `status`       | No                                                       | [`token`]     | A single or comma separated list of appointment statuses. Example: `arrived`
- [`_count`]     | No                                                       | [`number`]    | The maximum number of results to return.
+ Name                  | Required?                                                                          | Type          | Description
+-----------------------|------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------
+ `_id`                 | Yes, or one of `patient`, `practitioner`, or `location`.                           | [`token`]     | The logical resource id associated with the Appointment. Example: `3005759`
+ `date`                | Yes, or `-date-or-req-period` when using `patient`, `practitioner`, or `location`. | [`date`]      | The Appointment date time with offset. Example: `2019-06-07T22:22:16.270Z`
+ `-date-or-req-period` | Yes, or `date` when using `patient`, `practitioner`, or `location`.                | [`date`]      | The Appointment date time with offset. Example: `2019-06-07T22:22:16.270Z`
```

Comment:
this is searching by Appointment.start and Appointment.requestedPeriod.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/commit/72ae8eac60f6180eb4fdf81cf6184e39a2b326eb#diff-1d7df8ceb9f0ae8471323516e037c662R72-R73